indexer/services/comlink/src/controllers/api/v4/trades-controller.ts

```diff
@@ -62,6 +68,7 @@ class TradesController extends Controller {
           ? createdBeforeOrAtHeight.toString()
           : undefined,
         createdBeforeOrAt,
+        page,
       },
       [QueryableField.LIQUIDITY, QueryableField.CLOB_PAIR_ID, QueryableField.LIMIT],
```

Comment:
We need to specify a default ordering. Can you add
`{
        orderBy: [[FillColumns.eventId, Ordering.ASC]],
      },`?

Reply:
Done https://github.com/dydxprotocol/v4-chain/pull/1420/commits/2fd8d09f2b96b3cb0e71f79b02c10d7b8bba992e#diff-d8d0c764d2a842bf225a97d2160c105b481d115d5d483347392e0770ff1f404aR76